00-HTML-CSS-basics/css 2.1.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+
+    <div class="conteiner">
+
+      <div class="">
```

Comment:
I saw it later and I corrected in the last but I didn't remember if it was pushed